packages/styled-components/components/menu/menuItem.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as defaultStyles from './themes/default';
+
+/**
+ * Menu item.
+ *
+ * Render a single menu item.
+ *
+ * @todo Replace <a> with a proper <Link> component.
+ */
+const MenuItem = (props) => {
+  const {
+    attributeTitle,
+    classes,
+    children,
+    id,
+    level,
+    target,
+    theme = defaultStyles,
+    title,
+    url,
+  } = props;
+
+  const {
+    ItemWrapper,
+    Dropdown,
+  } = theme;
+
+  return (
+    <ItemWrapper key={id} classnames={classes}>
+      <li>
+        <a href={url} target={target} title={attributeTitle}>{title}</a>
```

Comment:
Title attributes aren't supported: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a
```suggestion
        <a href={url} target={target}>{title}</a>
```

Reply:
Thanks. Moved it to the <li> element.